scripts/parser-tests/flow/allowlist.txt

```diff
@@ -5,6 +5,7 @@ JSX/invalid_unpaired_rcurly.js
 JSX_invalid/migrated_0000.js
 arrow_function/object_return_type.js
 arrow_function_invalid/migrated_0002.js
+async_arrow_functions/with_type_parameters_types_disabled.js
```

Comment:
This test is added because Flow has a new test options: `types: boolean` that can control whether parameter types should be parsed: 

https://github.com/facebook/flow/blob/master/src/parser/test/flow/async_arrow_functions/with_type_parameters_types_disabled.options.json

We don't have such an option.

Reply:
From a parsing perspective, is it the same as `all: true`?